packages/webcomponentsjs/ts_src/platform/parent-node/append.ts

```diff
@@ -0,0 +1,37 @@
+/**
```

Comment:
Can you explain why these APIs are being added here rather than just in shadydom?

Reply:
This is to separate the polyfill for the feature itself from the part that adds support for the feature to Shady DOM. In this case (and for ChildNode), it was easier not to wrap the existing versions of these functions (built-in or polyfilled) since everything goes through Shady DOM's implementation of `insertBefore` and `removeChild` anyway, so Shady DOM doesn't actually use them. However, without them, we won't have support for these features without enabling Shady DOM.

Reply:
I'm just not sure this is worth doing since there is no browser that has Shadow DOM but does not have ParentNode/ChildNode.

Reply:
I didn't realize that there was any browser that had this configuration. It seems ok then.